DXRModules/DeusEx/Classes/DXRNames.uc

```diff
@@ -48,6 +48,10 @@ static function GiveRandomName(DXRando dxr, ScriptedPawn p)
 {
     if( p.bImportant || p.bIsSecretGoal ) return;
 
+    if (p.UnfamiliarName == "woman") {
```

Comment:
This fix should be in Fixups, not DXRNames?  Am I missing something?  Fixups will run before DXRNames does.